MalwareAnalyzer/views/domain_check.py

```diff
@@ -91,6 +91,24 @@ def malware_check(urllist):
 # Helper Functions
 
 
+def get_netloc(url):
+    """Get Domain."""
+    try:
+        domain = ''
+        parsed_uri = urlparse(url)
+        if not urlparse(url).scheme:
+            url = '//' + url
+            parsed_uri = urlparse(url)
+            domain = '{uri.netloc}'.format(uri=parsed_uri)
+        if (len(domain) > 2
+                and '.' in domain
+                and (domain.endswith('.') is False
```

Comment:
@superpoussin22 what is this check for?

Reply:
will try: on one side we have a list of url and in the other only one url

Reply:
@ajin: Rework down let me know what you think